tol_lab_share/config/rabbit.py

```diff
@@ -20,6 +22,20 @@
     vhost="tol",
 )
 
+# In our servers, this will be picked up using deployment project's
+# roles/deploy_tol_stack/templates/tol-lab-share/app_config.py.j2 and
+# environments/uat/group_vars/tol_swarm_managers.yml
+ENVIRONMENT_NAME = os.environ.get("ENVIRONMENT_NAME", "development")
```

Comment:
This seems to be quite specific to MLWH.  Should it be named more appropriately?